docs/lifecycle-methods.md

```diff
@@ -1,3 +1,7 @@
+<!--meta
+Documentation on Mithril's lifecycle methods/"hooks"
```

Comment:
This reads awkwardly and might confuse searches for "mithril lifecycle hooks". How about something like this?

```suggestion
Documentation on Mithril's lifecycle methods, also known as lifecycle "hooks"
```